conjur/logic/credential_provider/credential_store_factory.py

```diff
@@ -5,33 +5,62 @@
 
 This module is a factory for determining which credential store to use
 """
+# Third party
+import logging
```

Comment:
logging isn't third party?

Reply:
actually it is not part of the cpython Lib but I think that python installation come with it. so we can state it as builtin